packages/lesswrong/server/callbacks/postCallbackFunctions.tsx

```diff
@@ -1185,3 +1189,13 @@ export async function oldPostsLastCommentedAt(post: DbPost, context: ResolverCon
 
   await Posts.rawUpdateOne({ _id: post._id }, {$set: { lastCommentedAt: post.postedAt }})
 }
+
+export async function maybeCreateAutomatedContentEvaluation(post: DbPost, context: ResolverContext) {
+  const shouldEvaluate = isLW && !post.draft && !context.currentUser?.reviewedByUserId;
+  if (shouldEvaluate) {
```

Comment:
This will still evaluate the post on every update after it's published.  Maybe you want that?

Reply:
Oh, it's only for new users... hm.  I think checking `context.currentUser?.reviewedByUserId` is not actually safe and you should pull the user from the post's userId.

Reply:
why isn't it safe?